test/dataset_test.cpp

```diff
@@ -160,9 +160,9 @@ TEST(Dataset, get_variable_view) {
   EXPECT_EQ(d(Data::Value, "").name(), "");
   EXPECT_EQ(d(Data::Value, "name").tag(), Data::Value);
   EXPECT_EQ(d(Data::Value, "name").name(), "name");
-  EXPECT_THROW_MSG(d(Coord::Y), dataset::except::VariableNotFoundError,
-                   "Dataset with 3 variables, could not find variable with tag "
-                   "Coord::Y and name ``.");
+  EXPECT_THROW_MSG_SUBSTR(d(Coord::Y), dataset::except::VariableNotFoundError,
+                          "could not find variable with tag "
+                          "Coord::Y and name");
```

Comment:
Why can we not test the name any more?

Reply:
This was an attempt to stop the tests being over sensitive. i.e. small changes to formatting causing large numbers of tests to fail with little value. Personally I tend to test the type of the exception and not the message itself. Another alternative would be to expose the exception message strings themselves. If name is important, can add that back into the string check.

Reply:
By your argument you should then only test the exception type, and no text at all. After all the *name* is the only thing we have control over in this test.